lib/config.js

```diff
@@ -56,6 +56,7 @@ module.exports = {
     'allow-presentation',
     'allow-same-origin',
     'allow-scripts',
-    'allow-top-navigation'
+    'allow-top-navigation',
+    'allow-downloads-without-user-activation'
```

Comment:
A minor nitpick before merging: could we make sure this list is still alphabetized? I believe that will change the order here.

Reply:
I have fixed it. Thank you!